questions/css-box-model.md

```diff
@@ -0,0 +1,30 @@
+### Describe the layout of the CSS Box Model and briefly describe each component.
+
+#### Answer
+
+<!-- Your answer goes here. -->
+
+*Content*: The inner-most part of the box filled with text or an image or some type of content  
+
+*Padding*: The transparent area surrounding the content  
+
+*Border*: The area surrounding the padding (if any) and content  
+
+*Margin*: The transparent outer-most layer that surrounds the border
+
+#### Good to hear
+
+<!-- Whenever possible, include the short list of bullet points that sum up the answer. -->
+- This is a very common question asked during front-end interviews
```

Comment:
Come to think of it is this line needed? OF course it's common, that's why it's here isn't it?

Reply:
True, but I am thinking that maybe some people don't end up studying it before their interview because they think it's TOO easy, even though it is really important to know.  Maybe I can re-word it somehow?